_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,372 @@
+
+These instructions assume you have already created a free <a
+href="https://www.freenom.com" target="\_blank">domain through the freenom
+service</a>. If you have another domain, you can use that instead.
+
+Finally, setting up the SSL certificates on your EC2 instance requires you to use the terminal command line on your workstation. Make sure you have the `watch` command installed using the `which` command.
+
+```
+$ which watch
+/usr/local/bin/watch
+```
+
+If `watch` is not located, install it on your workstation.
+
+## Task 1: Launch an EC2 instance
+
+1. Visit the <a href="https://aws.amazon.com/free/" target="\_blank">AWS Free Tier page</a> and choose **Sign in to the Console**.
+
+   ![](/storage/images/aws-console.png)
+
+2. Make sure your region is set to `us-west-2` which is **Oregon**.
+
+   ![](/storage/images/us-west-2.png)
+
+3. Choose **Launch a virtual machine**.
+
+   The system opens the EC2 dashboard.
+
+4. Enter `Gaia` in the search bar.
+
+   The system finds AMIs in the Marketplace and the Community.
+
+5. Choose **Community AMIs**.
+
+   The system displays the available Gaia hub images.
+
+   ![](/storage/images/gaia-community.png)
+
+6. Select the **Blockstack.Gaia.Hub_1549643256 - ami-04793ff666176c1f3** image.
+
+   The system displays **Step 2: Choose an Instance Type** page.
+
+    ![](/storage/images/tier-2-image.png)
+
+7. Select **t2.micro** and choose **Configure Instance Details**.
+
+   1. Select a VPC.
+
+       A default VPC is created with a free tier account. You can use this
+       default VPC. Or you can choose another VPC. If you choose another VPC,
+       ensure the `Subnet` value is set to a subnet reachable by a public IP.
+
+       {% include important.html content="If you're using a private subnet, you
+       should attach an elastic IP (EIP) to the VM. This EIP allows you to
+       reboot the instance without worrying whether the address will reset. To
+       attach an IP, <strong>press allocate new address</strong> and follow the
+       instructions to attach the EIP to your new EC2 instance." %}
+
+   2. Set **Protect against accidental termination**.
+
+      If you terminate a Gaia instance, you lose all the data associated with it. Protection adds an extra step to terminating your Gaia instance.
+
+   3. Display the **Advanced Details**.
+
+      At this point, you are going to configure environment variables for your instance.  
+
+   4. Paste the following into the **Advanced Details**.
+
+      ```json
+      {
+        "ignition": { "version": "2.2.0" },
+        "storage": {
+          "files": [{
+            "filesystem": "root",
+            "path": "/etc/environment",
+            "mode": 420,
+            "contents": {
+              "source": "data:application/octet-stream,DOMAIN%3D<DOMAIN_NAME_VALUE>%0ASTAGING%3D<STAGING_VALUE>"
+            }
+          }]
+        }
+      }
+      ```
+
+      A value of `0` to requests a valid SSL certificate. At this point, you'll
```

Comment:
it's not clear at this point what the `0` is referring to -- I think we could just remove this sentence, since instructions are given in step 6.

Reply:
Fixed.